source/reference/release-notes.txt

```diff
@@ -20,11 +20,28 @@ Release Notes
 
 Learn what's new in:
 
+* :ref:`2.22 <ruby-version-2.22>`
 * :ref:`2.21 <version-2.21>`
 * :ref:`2.20 <version-2.20>`
 
 .. _upcoming-breaking-changes:
 
+.. _ruby-version-2.22:
+
+What's New in 2.22
+------------------
+
+The {+driver-short+} 2.21 release includes the following new features: 
+
+- Adds the ``type`` parameter to the
+  ``Mongo::SearchIndex::View#create_one`` and
+  ``Mongo::SearchIndex::View#create_many`` Search index
+  creation methods. This enables you to create Atlas Vector Search
```

Comment:
Q: Atlas Vector Search and Atlas Search?

Reply:
This feature specifically adds the functionality to create AVS indexes - the prev behavior only could create Search indexes.